### PR TITLE
Latest Profile Update at 21:20 on Jan 10

### PR DESCRIPTION
**Pull request by thebarshablog**

### CODE REVIEW

File: data/Pradumnasaraf.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "socials": [
     { "icon": "FaTwitter", "url": "https://twitter.com/pradumna_saraf" },
     { "icon": "FaGithub", "url": "https://github.com/Pradumnasaraf" },
-    { "icon": "website", "url": "https://pradumnasaraf.dev" }
+    { "icon": "FaLink", "url": "https://pradumnasaraf.dev" }
```
Comment (Panquesito7) [inline data/Pradumnasaraf.json:10]: I don't think you should update others' profiles 🤔 unless Pradumna accepts. Although it'd be better to tell the author (in this case, Pradumna) about it.

Comment (Pradumnasaraf) [reply data/Pradumnasaraf.json:10]: I checked you haven't synced your branches with the latest changes which is why it's reflecting old changed. Sync then you will good to go.

Screenshot 2023-01-10 at 10 07 00 PM

Comment (thebarshablog) [reply data/Pradumnasaraf.json]: > I checked you haven't synced your branches with the latest changes, which is why it's reflecting old changes. Sync then you will be good to go.
> 
> Screenshot 2023-01-10 at 10 07 00 PM

I have synced. Also, please review the events I have added. Once they're added o my profile, I'd delete my other branch...I'll keep me updated with the latest updates on brain branch.

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge.

**Review by Panquesito7 — APPROVED** [review]

**Comment by thebarshablog** [comment]
@Pradumnasaraf @Panquesito7 I'll delete my secondary branches once the changes have been merged and update my main branch.

**Pull request merged**